packages/neo-one-smart-contract-compiler/src/__data__/helpers/startNode.ts

```diff
@@ -0,0 +1,161 @@
+import {
+  ABI,
+  Client,
+  Contract,
+  DeveloperClient,
+  InvocationResult,
+  LocalKeyStore,
+  LocalMemoryStore,
+  LocalUserAccountProvider,
+  NEOONEProvider,
+  PublishReceipt,
+  SmartContract,
+  TransactionResult,
+  UserAccountID,
+} from '@neo-one/client';
+import { common, UInt160 } from '@neo-one/client-core';
+import { DiagnosticCategory, ts } from 'ts-simple-ast';
+import { createNode } from '../../../../neo-one-smart-contract-test/src/createNode';
+import { compile } from '../../compile';
+import { CompileResult } from '../../compile/types';
+import * as utils from '../../utils';
+
+// Is this needed here? if so...
```

Comment:
Remove the comments and open a card to consolidate the code between this file and setupTest.